mlflow/keras.py

```diff
@@ -61,9 +61,11 @@
 _PIP_ENV_SUBPATH = "requirements.txt"
 
 
-def _raise_deprecation_warning():
+def _raise_deprecation_warning(keras_module=None):
```

Comment:
@eedeleon We're planning to remove this function and drop support for keras < 2.3.0 and tensorflow < 2.0.

Reply:
Makes sense. However, currently I get a failure for 

keras==2.4.3
tensorflow==2.5.1

Would the correct PR be to remove it altogether?

Reply:
@eedeleon Thanks for the reply. Got it. If we `mlflow.keras._raise_deprecation_warning` that contains `import keras`, does that solve the issue?

Reply:
I think you mean remove. If so, yes

Reply:
Sorry, "remove" was missing.

Reply:
Would you like me to make a different PR removing the function and all calls to it? Or is someone else doing so.

Reply:
That'd be great!